python/paddle/jit/dy2static/utils.py

```diff
@@ -1478,7 +1478,7 @@ def _out_grad_names(program_desc, fwd_end_op_index, out_size):
         min(fwd_end_op_index + out_size, program_desc.block(0).op_size()),
     ):
         op = program_desc.block(0).op(i)
-        if op.type() == 'fill_any_like':
+        if op.type() in ('fill_any_like', 'fill_constant'):
```

Comment:
Add comment to show why we can need fill_constant here, better to add flag to show only in prim we need this.

Reply:
done